lib/cog/aggregate_command.rb

```diff
@@ -0,0 +1,57 @@
+require 'cog/exceptions'
+
+class Cog
+  class AggregateCommand < Cog::Command
+
+    def initialize
+      subcommands = self.class.const_get("SUBCOMMANDS")
+
+      unless subcommand = request.args.shift
+        no_subcommand_error
+        response.abort
+        return
```

Comment:
If we're raising an exception on line 10, will lines 11 and 12 ever be executed? Same goes for lines 16-18.